src/Components/ScrollToTop/ScrollToTop.css

```diff
@@ -0,0 +1,16 @@
+.scroll-top-btn {
+    position: fixed;
+    bottom: 3rem;
+    right: 1rem;
+    z-index: var(--z-index-4);
```

Comment:
Although this is the Component styling, I would suggest using better CSS variable names to make it more readable.